serverless/src/main.rs

```diff
@@ -30,8 +30,6 @@ pub async fn main() {
     Builder::from_env(env).target(Target::Stdout).init();
 
     let dd_api_key: Option<String> = env::var("DD_API_KEY").ok();
-    let dd_http_proxy: Option<String> = env::var("DD_HTTP_PROXY").ok();
```

Comment:
This was always wrong. `DD_HTTP(S)_PROXY` is used for proxying requests made by the agent for checks or integrations. To proxy the agents connection to Datadog, we've required `HTTPS_PROXY` be set.